### DISCUSSION TITLE
Relicensing

### DISCUSSION THREAD

**Discussion by inforithmics** [atifaziz/Fizzler#84] [Ideas]
Is it planned to relicense this library to a more permissive license? MIT, Apache ,BSD or similar?

**Comment by atifaziz** [comment]
There is no such plan at the moment because LGPL (_Lesser_ GPL) is already pretty permissive.

---
PS If this is with regards to SVG.NET, then see [my comment there on issue \#563](https://github.com/svg-net/SVG/issues/563#issuecomment-524795988).

**Comment by inforithmics** [comment]
What do you think about the MPL License which allows static linking.

https://softwareengineering.stackexchange.com/questions/179084/is-there-a-modified-lgpl-license-that-allows-static-linking
https://softwareengineering.stackexchange.com/questions/221365/mozilla-public-license-mpl-2-0-vs-lesser-gnu-general-public-license-lgpl-3-0
